SR2024/2023-12-13-challenge-reminder.md

```diff
@@ -0,0 +1,15 @@
+Hi $TODO_CONTACT_GIVEN_NAME,
+
+Just a reminder that the first challenge submission deadline passed over the weekend. 
+
+If you aren't aware of the challenges, you can find out more about them in the rulebook [here](https://studentrobotics.org/docs/resources/2024/rulebook.html#challenges)
+
+Congratulations to the 15 teams that managed to complete a challenge by that deadline.
+
+The next challenge deadline is 20th January 2024, so remember time will be short when your team gets back from Christmas break. There are another 5 league points up for grabs for anyone who manages to submit a challenge for this deadline.
+
+Your team can re-submit if their first attempt didn't pass, if you have any questions, just ask us via e-mail or Discord.
```

Comment:
This feels like it's two sentences to me?
```suggestion
Your team can re-submit if their first attempt didn't pass. If you have any questions, just ask us via e-mail or Discord.
```
It also feels like a bit more context might be useful?
```suggestion
Your team can re-submit if their attempts aren't judged to have passed the challenge, so it's worth having a go in plenty of time before the deadline. If you have any questions, just ask us via e-mail or Discord.
```